pkg/ovssubnet/controller/kube/bin/openshift-sdn-kube-subnet-setup.sh

```diff
@@ -3,13 +3,13 @@
 set -ex
 
 lock_file=/var/lock/openshift-sdn.lock
-subnet_gateway=$1
-subnet=$2
-cluster_subnet=$3
-subnet_mask_len=$4
-tun_gateway=$5
+local_subnet_gateway=$1
+local_subnet_cidr=$2
+local_subnet_mask_len=$3
+cluster_network_cidr=$4
+service_network_cidr=$5
```

Comment:
Yes (the commit message mentions that, although github tends to make you not notice commit messages in reviews...)